docs/howto/error-reporting.txt

```diff
@@ -194,6 +194,17 @@ filtered out of error reports in a production environment (that is, where
             def process_info(user):
                 ...
 
+    .. warning:: Use with ``asgiref.sync`` adapters
```

Comment:
`warning` won't render nice with a title. We can use `admonition` or remove the title.

Reply:
OK, dropped the title. Thanks 👍